applications/MPMApplication/custom_utilities/mpm_boundary_rotation_utility.h

```diff
@@ -499,14 +662,16 @@ class MPMBoundaryRotationUtility: public CoordinateTransformationUtils<TLocalMat
 private:
 	///@name Static Member Variables
 	///@{
-
-	const Variable<double>& mrFlagVariable;
+    constexpr static int SLIDING = 0;
+    constexpr static int STICK = 1;
```

Comment:
I recommend using an `enum class` for this, see `kratos/includes/global_variables.h`